saleor/graphql/product/mutations/products.py

```diff
@@ -1530,4 +1553,10 @@ def perform_mutation(cls, _root, info, image_id, variant_id):
             variant_image.delete()
 
         variant = ChannelContext(node=variant, channel_slug=None)
+
+        with transaction.atomic():
+            transaction.on_commit(
+                lambda: info.context.plugins.product_variant_deleted(variant)
+            )
```

Comment:
The same situation as above, IMO `product_variant_deleted` shouldn't be called for `VariantImageUnassign` mutation. Rather, `product_variant_updated`.

Reply:
Typo... 🙈